Gallery/Public/Item Implementations/ViewControllerItem.swift

```diff
@@ -25,33 +25,52 @@
 import UIKit
 
 public struct ViewControllerItem: Item {
-
+    
     // MARK: - Private properties
-
     private let viewControllerFactory: () -> UIViewController
-
+    
     // MARK: - Instantiation
-
     public init(title: String,
                 subtitle: String? = nil,
                 image: UIImage? = nil,
                 presentationStyle: PresentationStyle = .push,
-                viewControllerFactory: @escaping () -> UIViewController) {
+                viewControllerFactory: @escaping () -> UIViewController,
+                snapshot: Element.SnapshotTest? = nil) {
         self.title = title
         self.image = image
         self.subtitle = subtitle
         self.preferredPresentationStyle = presentationStyle
         self.viewControllerFactory = viewControllerFactory
+        if let snapshot = snapshot {
+            let vc = viewControllerFactory()
+            let view: UIView = vc.view
+            self.elementsProvider = ElementProvider(element: Element(title: String(describing: type(of: vc)) + "_" + title, view: view, width: 320, height: 568, snapshot: snapshot))
```

Comment:
I could, but I thought that I'd feel awkward, since we specify height explicitly here, and width is hardcoded somewhere inside. Do you recommend to leave it as default anyway?

Reply:
Yeah, I agree that it would not be great if you specify height but don't specify width. I'm thinking may be we should add default value the same way we have for width.

Reply:
👍

Reply:
I've created #7 for that